pollination/honeybee_vtk/translate.py

```diff
@@ -0,0 +1,50 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+
+
+@dataclass
+class Translate(Function):
+    """Translate hbjson file to vtkjs/vtp/vtk format."""
+
+    hbjson_file = Inputs.file(
+        description='HBJSON file.',
+        path='input.hbjson'
+    )
+
+    name = Inputs.str(
+        description='Name of the zip file.',
+        default='model'
+    )
+
+    file_type = Inputs.str(
+        description='Choose the type of files to export from "vtkjs", "vtp", "vtk"',
+        default='vtkjs'
```

Comment:
I would add a validation check here for this input. See here for an example:

https://github.com/pollination/honeybee-radiance/blob/7989fc54f157f386ffafde4efe5ee862db5fdf88/pollination/honeybee_radiance/rpict.py#L25-L26